packages/client/src/components/CreatePost.js

```diff
@@ -32,22 +31,13 @@ function CreatePost() {
   };
 
   const handleSubmit = async (e) => {
-    // we might want to go to the next page to see the published post (have to verify this)
     e.preventDefault();
-
-    setPost({ ...post, isDraft: e.target.value });
-
-    try {
-      const res = await fetch(endpoint, {
-        method: "POST",
-        headers: {
-          "Content-type": "application/json",
-        },
-        body: JSON.stringify({ post }),
-      });
-    } catch (error) {
-      console.log(error);
-    }
+    await axios.post(
+      `/api/post/create-post`,
+      { ...post, isDraft: e.target.value },
+      { headers: { Authentication: user.accesstoken } }
+    );
+    setRedirect(true);
```

Comment:
Is there a reason the `useHistory` doesn't work here?

Reply:
Sorry about that, switched it over.